src/common/BlogFrontmatter.res

```diff
@@ -1,42 +1,70 @@
-// Note: Every optional value in here must be encoded
-// as Js.Null.t, since it will be used for JSON serialization
-// within Next's static generation
-
-module Author = {
-  type t = {
-    username: string,
-    fullname: Js.null<string>,
-    role: string,
-    imgUrl: Js.null<string>,
-    twitter: Js.null<string>,
-  }
-
-  // We could *theoratically* query most of the
-  // author data from different sources, like twitter
-  // or github, but decided against it for simplicity
-  // reasons for now
-  @module("../../data/blog_authors.mjs") external rawAuthors: Js.Json.t = "default"
-
-  let getDisplayName = (author: t): string =>
-    switch author.fullname->Js.Null.toOption {
-    | Some(fullname) => fullname
-    | None => "@" ++ author.username
-    }
-
-  let decode = (json: Js.Json.t) => {
-    open Json.Decode
-    {
-      username: json->field("username", string, _),
-      fullname: json->optional(field("fullname", string), _)->Js.Null.fromOption,
-      role: json->field("role", string, _),
-      imgUrl: json->optional(field("img_url", string), _)->Js.Null.fromOption,
-      twitter: json->optional(field("twitter", string), _)->Js.Null.fromOption,
-    }
-  }
-
-  let getAllAuthors = (): array<t> => rawAuthors->Json.Decode.array(decode, _)
+type author = {
+  username: string,
+  fullname: string,
+  role: string,
+  imgUrl: string,
+  twitter: string,
 }
 
+let authors = [
```

Comment:
I've inlined this, which is a sliiight reduction in folks who don't know where to find this data. To be fair, it wasn't as discoverable previously in `data/` either. And it was weird to model it as an external we had to parse (??).

Reply:
Yeah makes sense, i just liked the fact that all the data is located in `data/` ... had all the dynamic data parts in one place, which is nice